src/controller/osw-controller.ts

```diff
@@ -16,10 +46,13 @@ class GtfsOSWController implements IController {
         this.intializeRoutes();
     }
 
+    eventBusService = new EventBusService();
+
     public intializeRoutes() {
         this.router.get(this.path, this.getAllOsw);
         this.router.get(`${this.path}/:id`, this.getOswById);
-        this.router.post(this.path, this.createOsw);
+        this.router.post(this.path, validationMiddleware(OswVersions), this.createOsw);
```

Comment:
remove this line. Its a duplicate implementation

Reply:
fixed